i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -488,6 +488,37 @@ Abhängig davon in welchem Status sich das Chat-Backup befindet, haben Sie zwei
 
 In beiden Fällen, stellen Sie sicher, dass Sie die erstellte Passphrase sicher aufheben.
 
+## Chat-Einstellungen
+
+### Verhalten der Eingabetaste ändern
+
+In diesem Abschnitt finden Sie eine Anleitung, wie Sie das Verhalten der Eingabetaste verändern können.
+
+:::info
+Diese Einstellung lässt sich für jedes Gerät einzeln vornehmen und wird auf die Standardeinstellung zurückgesetzt, wenn du dich über die Profileinstellungen abmeldest oder das Gerät in den Geräteeinstellungen entfernst. Nach der Anmeldung musst du die Einstellung dann erneut vornehmen.
```

Comment:
Probably here it would be better to say "unter „Verbundene Geräte“ in den Einstellungen" instead of "in den Geräteeinstellungen" since the "Devices" settings translate to "Verbundene Geräte" in the German Version.

Reply:
Thanks @mErrenst . Replaced.